src/main/kotlin/com/terraformation/backend/search/SearchFilter.kt

```diff
@@ -59,7 +97,7 @@ class NotNode(val child: SearchNode) : SearchNode {
   }
 }
 
-class FieldNode(
+data class FieldNode(
```

Comment:
Are `FieldNode`s the leaf nodes in the search?

Reply:
> Are `FieldNode`s the leaf nodes in the search?

Yep!